webrtc.js

```diff
@@ -65,6 +65,9 @@ var respecConfig = {
     },
     { name: "Taylor Brandstetter", company: "Google",
       w3cid: "82908"
+    },
+    { name: "Jan-Ivar Bruaroey", company: "Mozilla",
```

Comment:
Can we use ø here, or does he prefer to be Bruaroey?

Reply:
I took the same from the W3C records.

Reply:
@jan-ivar, any preference on this topic? :)

Reply:
I use Jan-Ivar Bruaroey.